tests/travis

```diff
@@ -1 +1 @@
-Subproject commit f62fc4fe97a88aa11e495abda384f142b38fd580
+Subproject commit b7a9cd1a1755b9f500ac111713ce07bb983c12e7
```

Comment:
Yes as it points to new travis version that recognizes the `UITEST_EXTRA_OPTIONS` variable